Formula/pymol.rb

```diff
@@ -81,9 +72,11 @@ def install
     args = %W[
       --install-scripts=#{libexec}/bin
       --install-lib=#{libexec}/lib/python#{xy}/site-packages
-      --pyqt PyQt5
+      --glut
+      --use-msgpackc c++11
     ]
     args << "--osx-frameworks" if OS.mac?
+    args << "--jobs 1" if ENV["CIRCLECI"] # Reduce memory usage for CircleCI.
```

Comment:
Next time that you update this formula, could you please try larger values, say `--jobs=4`?

Reply:
OK! I will do next time 😀